.eslintrc

```diff
@@ -802,7 +802,6 @@ rules:
       ignoreComments: true
       ignoreTrailingComments: true
       ignoreUrls: true
-      ignoreStrings: true
```

Comment:
Mentioned in chat… I would revert this change and keep `ignoreStrings: true` since that is the default, but having it here shows the behavior without needing to look it up in the docs.

Reply:
I added ignoreStrings back into the main .eslintrc. There were more max-len lint warnings when I removed ignoreStrings then when it was set to true. It seemed like the default setting was false.